verilator-config.cmake.in

```diff
@@ -51,6 +51,12 @@ endif()
 
 set(verilator_FOUND 1)
 
+find_program(CCACHE_FOUND ccache)
+
+if(CCACHE_FOUND)
+    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
+endif()
+
```

Comment:
Thanks!

Presumably this should be reconciled with this code:  https://github.com/verilator/verilator/blob/84f26ed3d7c7a6bfeb1c39c1fcede4369d2a6e92/verilator-config.cmake.in#L314-L343

perhaps replacing the CMAKE_CXX_COMPILER_LAUNCHER line?

Reply:
I'll take look on that over weekend.

Reply:
I looked into CMAKE_CXX_COMPILER_LAUNCHER - it is set to proper value but somehow is ignored by cmake. So I've changed that piece to global RULE_LAUNCH_COMPILE. I think it is not right way but it is working solution. Since it is optional I think it should be pretty safe to include that tweak into the next release - users will always have ability to disable ccache via `OBJCACHE_ENABLED = OFF` in case if they spot local troubles.

Honestly I'm not cmake expert - I use and hate it for 20 years already. Many years ago I moved to `gyp` and then to `gn`. I'm finally happy to use something built by humans for mere mortals.

Perhaps someone with strong passion and love to cmake will fix it in a better way.